expression/column.go

```diff
@@ -189,8 +188,7 @@ func (col *Column) String() string {
 
 // MarshalJSON implements json.Marshaler interface.
 func (col *Column) MarshalJSON() ([]byte, error) {
-	buffer := bytes.NewBufferString(fmt.Sprintf("\"%s\"", col))
-	return buffer.Bytes(), nil
+	return []byte(fmt.Sprintf("\"%s\"", col)), nil
```

Comment:
why not just calling `col.String()` ?

Reply:
Because there is double quote around it.